FORMULA/crowdin.rb

```diff
@@ -0,0 +1,16 @@
+class Crowdin < Formula
+  desc "Crowdin-CLI is a command-line tool that allows to manage your resources"
+  homepage "https://support.crowdin.com/cli-tool/"
+  url "https://crowdin.com/downloads/crowdin-cli-2.0.11.zip"
+  sha256 "027925176948b8973ba1a9fab4f0c097bf49f2df0c23be20d79049bd7af4fbc3"
+
```

Comment:
This needs a `depends_on :java` (or `depends_on :java => "1.8+"` if there is a minimum version requirement).